src/case-ui-toolkit.module.ts

```diff
@@ -107,7 +107,6 @@ import { CaseHistoryModule } from './shared/components/case-history';
       WriteNumberFieldComponent,
       WriteEmailFieldComponent,
       WriteDateFieldComponent,
-      WriteCaseFlagFieldComponent,
```

Comment:
@johnbenjamin-hmcts - WriteCaseFlagFieldComponent is no longer being exported... is that intentional? (Maybe it doesn't need to be exported.)